pierce-portfolio/portfolio/src/main/webapp/fotos.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+   <head>
+    <meta charset="UTF-8">
+    <title>Pierce D. Wilson</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+ <div class="topnav">
+    <a href="/">About me</a>
+    <a href="/resume.html">Resume</a>
+    <!-- <a href="/projects.html">Projects</a> -->
+    <!-- <a href="/writing.html">Writing</a> -->
+    <a class="active" href="/fotos.html">Photos!</a>
+
+<body>
+    <div id="content">
+        <br><br>
+    <h1>My life in photos.</h1>
+      <h3>These are some pictures that show a little bit about my life. Check back for more later!</h3>
+      <p>Me at the beach on the coast of New Orleans.</p>
+      <img src="images/beach.jpg" width="500" height="600" />
```

Comment:
Since you're re-using this styling for all the images, you can create a style class and just set "class=" to all the imgs.